channels/candidate-4.2.yaml

```diff
@@ -52,3 +52,11 @@ versions:
 - 4.2.21
 - 4.2.22
 - 4.2.23
+tombstones:
```

Comment:
Is there a reason 4.2.11 and 4.2.11-s390x were not included in the tombstones? We wouldn't want those promoted either right?

Reply:
Also I'd expect a reason / BZ for any release we choose to tombstone

Reply:
> Is there a reason 4.2.11 and 4.2.11-s390x were not included in the tombstones? We wouldn't want those promoted either right?

We had already promoted those before we noticed the problem:

```console
$ grep -A1 '4\.2\.11' channels/*-4.2.yaml 
channels/candidate-4.2.yaml:# - 4.2.11 failed to run tests, we never officially released it, but we accidentally
channels/candidate-4.2.yaml-# put it in a channel! (same for s390x)   Now we shouldn't pull it, just in case
--
channels/candidate-4.2.yaml:- 4.2.11
channels/candidate-4.2.yaml:- 4.2.11-s390x
channels/candidate-4.2.yaml-- 4.2.12
--
channels/fast-4.2.yaml:# - 4.2.11 failed to run tests, we never officially released it, but we accidentally
channels/fast-4.2.yaml-# put it in a channel! (same for s390x)   Now we shouldn't pull it, just in case
--
channels/fast-4.2.yaml:- 4.2.11
channels/fast-4.2.yaml:- 4.2.11-s390x
channels/fast-4.2.yaml-- 4.2.12
--
channels/stable-4.2.yaml:# - 4.2.11 failed to run tests, we never officially released it, but we accidentally
channels/stable-4.2.yaml-# put it in a channel! (same for s390x)   Now we shouldn't pull it, just in case
--
channels/stable-4.2.yaml:- 4.2.11
channels/stable-4.2.yaml:- 4.2.11-s390x
channels/stable-4.2.yaml-- 4.2.12
```

> Also I'd expect a reason / BZ for any release we choose to tombstone

I've been adding that for new tombstones (e.g. c69dc22d241), but hadn't gone back through internal discussion to see why other releases were tombstoned.  Should I do that digging?